### ISSUE TITLE
missing commands for completion of package tkz-euclide 

### ISSUE THREAD

**Issue by DailyShana** [James-Yu/LaTeX-Workshop#4257]
I find many command in tkz-euclide doesn't have a completion. I compare the content of https://github.com/James-Yu/LaTeX-Workshop/blob/d82ba4951aa606185d12fb6e02ef871bb330af3d/data/packages/tkz-euclide.json and https://github.com/texstudio-org/texstudio/blob/337e247ecb1b609b5eb07b3a319973c55d6fd4d0/completion/tkz-euclide.cwl.
I find the json generated here missing many command:
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/8469304/3eca20bd-3240-43ca-8e8b-144d7bcbeea4)
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/8469304/a7f6af2a-1ab9-4211-9850-90935d155586)

**Comment by James-Yu** [comment]
We don't parse commands with `()` as arguments. It will cause more trouble in auto-suggestion than convenience.

**Comment by DailyShana** [comment]
if it's hard to parse arguments in `()`, just record the name of the commands. Hint of the command without arguments is still useful.
